services/order_by.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.utils.translation.trans_real import translation
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list[dict], username: str, date: datetime) -> list[dict]:
+    with translation.atomic():
+        order = Order.objects.create(username=username, created_at=date)
```

Comment:
The `Order` model does not have a `username` field according to the `db/models.py` file. It should be `user` and you need to pass a `User` instance instead of a username string.